java/org/contikios/cooja/contikimote/ContikiMoteType.java

```diff
@@ -185,6 +185,12 @@ public String getMoteType() {
     return "cooja";
   }
 
+  @Override
+  public String getMoteTypeIdentifierPrefix() {
+    // The "mtype" prefix for ContikiMoteType is hardcoded elsewhere, so use that instead of "cooja".
+    return "mtype";
```

Comment:
Where is it hardcoded elsewhere? I think this is the only remaining place, and we could skip this special handling of Cooja motes if we want to. I think the only difference of removing this special handling would be that the library files would be named "cooja<number>.cooja" instead of "mtype<number>.cooja". What do you think?

Reply:
I would like to get rid of this special case, but I think it should be done in a separate PR after the csc file updates are merged in Contiki-NG.